app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    if command.split()[0] == "mv" and not command.endswith("/"):
```

Comment:
It is usually considered best practice not to put function's main logic into an `if` condition. I'd suggest to rewrite it like this:
```python
if ...:
    return

...

Reply:
I wanted to do this, but I thought it would be wrong

Reply:
Look at your code now: it's immediately clear that the function terminates when the `if` block executes, so the reader doesn't have to look any further to double check.

Aside from that, there was an extra layer of indentation that is now gone. This is also pretty good. :)